incubator/hnc/pkg/reconcilers/hnc_config_test.go

```diff
@@ -12,14 +13,24 @@ import (
 	v1 "k8s.io/api/rbac/v1"
 	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
 var _ = Describe("HNCConfiguration", func() {
-	// sleepTime is the time to sleep for objects propagation to take effect.
+	// objectPropagationTime is the time to sleep for objects propagation to take effect.
 	// From experiment it takes ~0.015s for HNC to propagate an object. Setting
 	// the sleep time to 1s should be long enough.
 	// We may need to increase the sleep time in future if HNC takes longer to propagate objects.
-	const sleepTime = 1 * time.Second
+	const objectPropagationTime = 1 * time.Second
```

Comment:
We should specify that we only use this when we're expecting something _not_ to happen. Otherwise, we just use Eventually.

Reply:
Done. Thanks for the suggestion!